### PR TITLE
fix(deprecations): Fix deprecation warnings

### PR DESCRIPTION
**Pull request by vishaltelangre**
Fixes the following deprecation warnings when used in an Elixir v1.9 project.

1. `warning: String.replace/4 with :insert_replaced option is deprecated. Please use :binary.replace/4 instead or pass an anonymous function as replacement`
2. `warning: clauses with the same name should be grouped together, "def analyze/1" was previously defined`

### CODE REVIEW

File: lib/veritaserum.ex [hunk 1]
```diff
@@ -112,7 +112,7 @@ defmodule Veritaserum do
     |> String.replace(~r/\n/, " ")
     |> String.downcase()
     |> String.replace(~r/[.,\/#!$%\^&\*;:{}=_`\"~()]/, " ")
-    |> String.replace(Evaluator.emoticon_list(), "  ", insert_replaced: 1)
+    |> :binary.replace(Evaluator.emoticon_list(), "  ", insert_replaced: 1)
```
Comment (vishaltelangre) [inline lib/veritaserum.ex:115]: Fixes the following deprecation warning.

```
warning: String.replace/4 with :insert_replaced option is deprecated. Please use :binary.replace/4 instead or pass an anonymous function as replacement
  (elixir) lib/string.ex:1373: String.replace/4
  (veritaserum) lib/veritaserum.ex:115: Veritaserum.clean/1
  (veritaserum) lib/veritaserum.ex:51: Veritaserum.get_list_with_marks/1
  (veritaserum) lib/veritaserum.ex:31: Veritaserum.analyze/2
  (veritaserum) lib/veritaserum.ex:38: Veritaserum.analyze/1
```

File: lib/veritaserum.ex [hunk 2]
```diff
@@ -17,6 +17,12 @@ defmodule Veritaserum do
     |> Enum.to_list()
   end
 
+  def analyze(input) do
+    {score, _} = analyze(input, return: :score_and_marks)
+
+    score
+  end
+
```
Comment (vishaltelangre) [inline lib/veritaserum.ex:25]: Fixes following deprecation warning.

```
warning: clauses with the same name should be grouped together, "def analyze/1" was previously defined (lib/veritaserum.ex:14)
  lib/veritaserum.ex:37
```

File: .travis.yml [hunk 3]
```diff
@@ -3,3 +3,4 @@ elixir:
   - 1.3.4
   - 1.4.5
   - 1.5.1
+  - 1.9.0
```
Comment (vishaltelangre) [inline .travis.yml:6]: I am not sure if this is required as part of this changeset. Some of the older versions, as well as the recently released 1.10.0 version, is missing from this list.

Comment (uesteibar) [reply .travis.yml:6]: I'll review the versions here when I have some time, meanwhile this addition is great 👍

**Review by uesteibar — APPROVED** [review]
Thanks for taking care of this! 👍

**Pull request merged**